modules/custom/az_mail/az_mail.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Quickstart Mail
+type: module
+description: Send mail via SMTP
```

Comment:
Do you think we should make this description a little more specific?  Maybe, "Send mail via SMTP using Amazon SES."

A README with some directions for how to use/configure the module would probably be a good future improvement.